### ISSUE TITLE
New model for simple glass

### ISSUE THREAD

**Issue by mbonvini** [lbl-srg/modelica-buildings#425]
The construction type `OpenStudio::Model::SimpleGlazing` 
https://openstudio-sdk-documentation.s3.amazonaws.com/cpp/OpenStudio-1.7.0-doc/model/html/classopenstudio_1_1model_1_1_simple_glazing.html 

cannot be implemented because there's not a counterpart in the Buildings library.

This simple model for glazing systems is based on a U-Factor, solar heat gain coefficient and visible transmittance.

**Comment by mwetter** [comment]
@lambtt : You would need to take a window system from https://simulationresearch.lbl.gov/modelica/releases/v9.1.0/help/Buildings_HeatTransfer_Data_GlazingSystems.html#Buildings.HeatTransfer.Data.GlazingSystems or build your own window system that has similar properties than the `SimpleGlazing`.

**Comment by mwetter** [comment]
Yes, unless the source of the `SimpleGlazing` states what window it is in reality, this requires some trial and error to come up with a construction that has similar properties.
